spec/behavior/httpEnvelope.spec.js

```diff
@@ -664,6 +664,37 @@ describe( 'HTTP Envelope', function() {
 				} );
 			} );
 		} );
+
+		describe.only( 'with body without hasOwnProperty', function() {
```

Comment:
@calvinb - :notes: `.only` is the loneliest `describe` that you'll ever do ... :notes: